inst/include/dust/dust.hpp

```diff
@@ -394,6 +36,9 @@ class Dust {
     _rng(_n_particles_total, seed),
     _stale_host(false),
     _stale_device(true) {
+#ifdef __NVCC__
+    initialise_device(0); // TODO - set this from value in constructor
```

Comment:
Shall we wire the device ID up now?

Reply:
I think we have all the bits, yeah. I'll do that tomorrow morning